### PR TITLE
Removing refunds for `SELDESTRUCT`'s

### PR DESCRIPTION
**Pull request by OlivierBBB**

### CODE REVIEW

File: hub/constraints/generalities/gas.lisp [hunk 1]
```diff
@@ -16,9 +16,7 @@
                (begin (hub-stamp-constancy GAS_EXPECTED)
```
Comment (lorenzogentile404) [inline hub/constraints/generalities/gas.lisp:16]: You may use the alias here to be completely consistent with specs. It already exists in the `lisp`, but it is not currently used:

```
(defalias
    ;;
   ...
    GAS_XPCT            GAS_EXPECTED
```

File: hub/constants.lisp [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 (defconst
   MULTIPLIER___STACK_HEIGHT                8      ;; for height increments
-  MULTIPLIER___DOM_SUB_STAMPS             16      ;; for dom/sub stamp business
-  DOM_SUB_STAMP_OFFSET___REVERT            8
-  DOM_SUB_STAMP_OFFSET___SELFDESTRUCT     12
+  MULTIPLIER___DOM_SUB_STAMPS              8      ;; for dom/sub stamp business
+  DOM_SUB_STAMP_OFFSET___REVERT            6
+  DOM_SUB_STAMP_OFFSET___SELFDESTRUCT      7
   )
```
Comment (lorenzogentile404) [inline hub/constants.lisp:8]: I guess these correspond to $\lambda$, $\epsilon_{rev}$ and $\epsilon_{selfdest}$ in the specs. I believe we should at least add a comment make it clear what they are referring to in the specs.

**Review by lorenzogentile404 — CHANGES_REQUESTED** [review]

File: hub/constants.lisp (hunk 2 above)
Comment (OlivierBBB) [reply hub/constants.lisp:8]: Done

File: hub/constraints/generalities/gas.lisp (hunk 1 above)
Comment (OlivierBBB) [reply hub/constraints/generalities/gas.lisp:16]: Do you want to add an alias ?

Comment (lorenzogentile404) [reply hub/constraints/generalities/gas.lisp:16]: In the constraints.lisp file the extended version `GAS_EXPECTED` is used, even if the alias `GAS_XPCT` has been created (that is the same notation we use in the spec).

Comment (OlivierBBB) [reply hub/constraints/generalities/gas.lisp:16]: I see

Comment (OlivierBBB) [reply hub/constraints/generalities/gas.lisp:16]: See https://github.com/Consensys/linea-constraints/issues/330

**Review by lorenzogentile404 — APPROVED** [review]

**Pull request merged**